tools/DeviceExplorer/DeviceExplorer/DevicesProcessor.cs

```diff
@@ -28,50 +28,33 @@ public DevicesProcessor(string iotHubConnenctionString, int devicesCount, string
             try
             {
                 DeviceEntity deviceEntity;
-                var devices = await registryManager.GetDevicesAsync(maxCountOfDevices);
+                var query = registryManager.CreateTwinQuery(null);
```

Comment:
Please add the type for `query` instead of `var`. (var not allowed if the type is not present on the same line.)